apps/music/views/playlists/view.js

```diff
@@ -49,34 +49,32 @@ var PlaylistsView = View.extend(function PlaylistsView() {
 
   this.client.on('databaseChange', () => this.update());
 
-  this.update();
+  this.update()
+    .then(() => this.list.scrollTo(this.searchBox.HEIGHT));
```

Comment:
I can kinda see why you might want to have a promise return from `update()` now. You can ignore my comments regarding return values. However, in this case, I would just put this `scrollTo()` call directly in the `update()` method.